examples/elastigroup.yml

```diff
@@ -0,0 +1,61 @@
+SenzaInfo:
+  StackName: hello-world
+  StackVersion: one
+
+SenzaComponents:
+  - AppServerElastigroup:
+      Type: Spotinst::Elastigroup
+      "ServiceToken": "arn:aws:lambda:us-west-2:178579023202:function:spotinst-cloudformation"
+      "accessToken": "spotinst-token"
+      Elastigroup:
+        name: Spotinst-CloudFormation
+        strategy:
+          risk: 100
+          availabilityVsCost: balanced
+        capacity:
+          target: 1
+          minimum: 1
+          maximum: 1
+        scaling: {}
+        compute:
+          instanceTypes:
+            ondemand: m3.large
+            spot:
+            - m3.large
+            - m4.large
+            - c3.large
+            - c4.large
+          availabilityZones:
```

Comment:
Could this also work with the [auto-configuration](http://stups.readthedocs.io/en/latest/components/senza.html#senza-stupsautoconfiguration) (discovery of subnets)